guava-tests/test/com/google/common/collect/ImmutableSetTest.java

```diff
@@ -582,4 +582,12 @@ private static long worstCaseQueryOperations(Set<?> set, CallsCounter counter) {
     }
     return worstCalls;
   }
+
+  public void testReuseBuilderReducingHashTableSizeWithPowerOfTwoTotalElements() {
+    final ImmutableSet.Builder<Object> builder = ImmutableSet.builderWithExpectedSize(6);
```

Comment:
We don't typically include `final` as you do here, considering it redundant -- it does happen to be final, but that has no real importance to the correctness of the test.